commands.c

```diff
@@ -373,6 +373,7 @@ int mutt_display_message(struct MuttWindow *win_index, struct MuttWindow *win_ib
     struct PagerView pview = { &pdata };
 
     pdata.email = e;
+    pdata.msg = msg;
```

Comment:
We'll need to set this to `NULL`, when we later:
```c
mx_msg_close(m, &msg);
```

(I can't add comments to code you haven't changed :-)